src/services/Subscriptions.php

```diff
@@ -280,4 +284,39 @@ public function getUnsavedDraftByUid(StripeSubscription $subscription): Subscrip
             ->drafts()
             ->one() ?? new SubscriptionElement();
     }
+
+    /**
+     * Ensures that a user with given email address is created if one doesn't already exist.
+     *
+     * @param StripeSubscription $subscription
+     * @param SubscriptionElement $subscriptionElement
+     * @return void
+     * @throws \yii\base\Exception
+     * @throws \yii\base\InvalidConfigException
+     */
+    private function maybeEnsureUser(StripeSubscription $subscription, SubscriptionElement $subscriptionElement): void
+    {
+        $plugin = Plugin::getInstance();
+        $settings = $plugin->getSettings();
+
+        if ($settings['createUserIfMissing']) {
```

Comment:
```suggestion
        if ($settings->createUserIfMissing) {
```

Might be worth flipping this to the inverse with an "exit early" strategy.

Reply:
Tweaked this, but the previous change means exiting early is no longer needed.